WordPress/Classes/Utility/BuildInformation/RemoteConfig.swift

```diff
@@ -47,4 +47,8 @@ struct RemoteConfig {
     var blazeFlowCompletedStep: RemoteConfigParameter<String> {
         RemoteConfigParameter<String>(key: "blaze_completed_step_hash", defaultValue: "step-5", store: store)
     }
+
+    var wordPressPluginOverlayMaxShown: RemoteConfigParameter<String> {
```

Comment:
Does the remote config support an `Int` type?

I haven't used it yet and looking at the rest of this file, everything is a `String`. We don't have to change this to an `Int` if it may cause an issue. I was just curious about it.

Reply:
I've just tested through my sandbox that `Int` types are supported from the backend and the iOS client, and it looks like it's supported in Android too. Changed the config type to `Int` in https://github.com/wordpress-mobile/WordPress-iOS/pull/20389/commits/60ced615d3ee1ae7a215d3bbe31fd4db22ae0120. 🙂 

p1679651395096549-slack-C04LSQ95WTW